pystiche_papers/gatys_et_al_2017/_loss.py

```diff
@@ -49,7 +49,7 @@ def __init__(
             layer_weights=layer_weights,
             score_weight=score_weight,
         )
-
+        # https://github.com/pmeier/NeuralImageSynthesis/blob/cced0b978fe603569033b2c7f04460839e4d82c4/LossLayers.lua#L80
```

Comment:
Again, I fail to see how this line relates to the `score_correction_factor`. With the same argument as above this is a combination of [`L63`](https://github.com/pmeier/NeuralImageSynthesis/blob/cced0b978fe603569033b2c7f04460839e4d82c4/LossLayers.lua#L63) and [`L75`](https://github.com/pmeier/NeuralImageSynthesis/blob/cced0b978fe603569033b2c7f04460839e4d82c4/LossLayers.lua#L75)